src/main.ts

```diff
@@ -1,5 +1,71 @@
+import * as os from "os"
+
 import { ExtensionContext } from "vscode";
+import { Config } from "./config";
+import { createLanguageClient } from "./client";
+import { LanguageClient } from "vscode-languageclient";
+
+let client: LanguageClient | null;
+
+/**
+ * Called when the plugin is activated
+ * @param context The vscode context for this extension
+ */
+export async function activate(context: ExtensionContext) {
+
+    const config = new Config(context);
+
+    // Bootstrap the extension, this returns an optional Mun executable path which we can use to 
+    // initialize a language server with
+    const executablePath = await bootstrap(config);
+    if (executablePath != null) {
```

Comment:
Maybe log or throw a pop-up error to signal failure?

Reply:
This is not a failure case. I don't want to advertise this too much yet. I was planning on adding a warning when we have something useful.

Reply:
Then maybe at least add a `TODO` comment to prevent it from being forgotten

Reply:
Done